### PR TITLE
feat:proposal page

### PR DESCRIPTION
**Pull request by florashore**

### CODE REVIEW

**Review by coderprovider — COMMENTED** [review]
good proposal page

**Comment by florashore** [comment]
🙏

**Comment by coderprovider** [comment]
what about holders?

**Comment by florashore** [comment]
user count to use the LTV

**Review by coderprovider — COMMENTED** [review]
is unsaved

**Comment by florashore** [comment]
right
will fix

**Comment by florashore** [comment]
fixed

**Review by coderprovider — COMMENTED** [review]
cool

File: src/pages/proposalDetail/proposalDetail.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+import { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import ProposalShow from "../../components/proposalShow/proposalShow";
+import Information from "../../components/Information/information";
+import ButtonTemplate from "../../components/button/button";
+import RangeInput from "../../components/rangeInput/rangeInput";
+import axios from "axios";
+
+const ProposalDetail = () => {
```
Comment (miraclaers) [inline src/pages/proposalDetail/proposalDetail.js:11]: good

File: src/pages/proposalDetail/proposalDetail.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                  <p className="text-[16px] font-semibold">
+                    {oneProData[0]?.title}
+                  </p>
+                  <p className="text-[12px] font-semibold mt-2" >
+                    {oneProData[0]?.optional}
+
+                  </p>
+                  <div className="mt-[20px]">
+                    <p className="text-[12px] font-normal">
+                      {oneProData[0]?.description}
```
Comment (miraclaers) [inline src/pages/proposalDetail/proposalDetail.js:92]: ?

File: src/pages/proposalDetail/proposalDetail.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                    <span className="text-[10px] text-[#9B9B9B] font-bold">
+                      Your voting power :
+                    </span>
+                    <span className="text-[10px] text-[#9B9B9B] font-semibold">
+                      6,500 $TOKEN
+                    </span>
+                  </div>
+                  <ButtonTemplate
+                    onclick={startVote}
+                    buttonStyle="ml-[-100px] "
```
Comment (miraclaers) [inline src/pages/proposalDetail/proposalDetail.js:118]: ?

**Review by miraclaers — COMMENTED** [review]
?

**Comment by florashore** [comment]
is z-index

**Comment by florashore** [comment]
?

**Comment by miraclaers** [comment]
understood

**Review by miraclaers — COMMENTED** [review]
❤

File: src/pages/proposalDetail/proposalDetail.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+      getOneProposal()
+    }
+  }, [res])
+
+  const startVote = async () => {
+
+    const res = await axios.post(`${process.env.REACT_APP_API_URL}/getproposal/getnumber`, { proposalId })
+
+    if (contract) {
+      const contractId = res?.data[0]?.proposalId / 1
```
Comment (Jermaine522) [inline src/pages/proposalDetail/proposalDetail.js:50]: unnecessay

**Review by Jermaine522 — COMMENTED** [review]
usestate is wrong

File: src/pages/proposalDetail/proposalDetail.js (hunk 4 above)
Comment (florashore) [reply src/pages/proposalDetail/proposalDetail.js:50]: will fix it

**Comment by Jermaine522** [comment]
others good

**Comment by florashore** [comment]
🙏

**Pull request merged**